oauth2/oauth2-core/src/test/java/org/springframework/security/oauth2/core/user/OAuth2UserAuthorityTests.java

```diff
@@ -58,4 +60,11 @@ public void constructorWhenAllParametersProvidedAndValidThenCreated() {
 		assertThat(userAuthority.getAttributes()).isEqualTo(ATTRIBUTES);
 	}
 
+	@Test
+	public void convertURLReplacesURLWithString() throws MalformedURLException {
```

Comment:
Please remove this test and add a test for `equals()` and `hashCode()` using 2 different instances of `OAuth2UserAuthority`